packages/multi-test/src/wasm.rs

```diff
@@ -25,20 +25,6 @@ use cosmwasm_std::testing::mock_wasmd_attr;
 
 use anyhow::{bail, Context, Result as AnyResult};
 
-// TODO: we should import this from cosmwasm-std, but cannot due to non_exhaustive so copy here
-#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
-pub struct ContractInfoResponse {
```

Comment:
This `pub` was never used, right? At least I did not find something.

Reply:
It was not. Maybe pub for test cases?

Reply:
Probably just copy/paste from the cosmwasm repo